Sources/SwifterSwift/Foundation/DateExtensions.swift

```diff
@@ -43,6 +43,27 @@ public extension Date {
         /// SwifterSwift: Full month name.
         case full
     }
+    
+    /// SwifterSwift: we have three predefined cases (fullDateTime, shortDate, time) that represent commonly used date formats.
+    enum DateFormat {
+        case fullDateTime
+        case shortDate
+        case time
+        case custom(String)
+        
+        var getString: String {
+            switch self {
+            case .fullDateTime:
+                return "yyyy-MM-dd HH:mm:ss"
```

Comment:
It's location-specific, so I this code isn't universal. As so I don't think it's a very good idea for an extension for people around the world to use. The only good way to make this extension universal is to rename cases of the enum to something like `USFullDateTime`, `europeanFullDateTime` etc.